CHANGELOG.md

```diff
@@ -5,6 +5,73 @@ All notable changes to this project will be documented in this file.
+
+### Changed
+
+- Hiding _Disable AMP_ field on settings page when the AMP plugin is not enabled. [#519](https://github.com/Parsely/wp-parsely/pull/519)
+- Use built-in WordPress submit button instead of custom one in the settings page. [#513](https://github.com/Parsely/wp-parsely/pull/513)
+- Improved wp-admin settings page help texts. [#552](https://github.com/Parsely/wp-parsely/pull/552)
+- `@wordpress/scripts` bumped from 19.2.1 to 19.2.3. [#503](https://github.com/Parsely/wp-parsely/pull/503) [#603](https://github.com/Parsely/wp-parsely/pull/603)
+- `prettier` bumped from 2.4.1 to 2.5.0. [#509](https://github.com/Parsely/wp-parsely/pull/509)
+- `concurrently` bumped from 6.4.0 to 6.5.1. [#551](https://github.com/Parsely/wp-parsely/pull/551)
+- Ubuntu bumped from 18.04 to 20.04 on CI tests. [#445](https://github.com/Parsely/wp-parsely/pull/445)
+- Unit and Integration tests run in random order. [#511](https://github.com/Parsely/wp-parsely/pull/511)
+- Correct Parse.ly spelling in tests comments. [#561](https://github.com/Parsely/wp-parsely/pull/561)
+- Minor amendments on the documentation. [#514](https://github.com/Parsely/wp-parsely/pull/514)
+- Updated release process guidelines. [#567](https://github.com/Parsely/wp-parsely/pull/567)
+- Removed checkboxes from GitHub's PR template. [#512](https://github.com/Parsely/wp-parsely/pull/512)
+- Improved JS scripts integration tests. [#557](https://github.com/Parsely/wp-parsely/pull/557)
+- Source code linting violations (PHPCS with `--serverity=1`). [#544](https://github.com/Parsely/wp-parsely/pull/544)
+- WordPress.org screenshots for settings page. [#574](https://github.com/Parsely/wp-parsely/pull/574)
+
+### Fixed
+
+- Undefined index error on settings page. [#536](https://github.com/Parsely/wp-parsely/pull/536)
+- Source the correct asset for the Recommendations Widget. [#545](https://github.com/Parsely/wp-parsely/pull/545)
+- End-to-end tests in CI (GitHub Actions). [#521](https://github.com/Parsely/wp-parsely/pull/521)
+
+### Removed
+
+- Plugin version number being printed in front-end's HTML source code. [#502](https://github.com/Parsely/wp-parsely/pull/502)
+- Custom CSS on wp-admin settings page. [#496](https://github.com/Parsely/wp-parsely/pull/496)
+- `migrate_old_fields` private function from Recommended Widget. [#599](https://github.com/Parsely/wp-parsely/pull/599)
+- PHP 8.2 from CI tests. [#523](https://github.com/Parsely/wp-parsely/pull/523)
+- Custom end-to-end Docker image. [#524] (https://github.com/Parsely/wp-parsely/pull/524)
+
+
+## [3.0.4] - 2022-01-17
+
+### Changed
+
+- Changed plugin loading functions from anonymous to named functions. [#595](https://github.com/Parsely/wp-parsely/pull/595)
+
+## [3.0.3] - 2022-01-12
+
+### Fixed
+
+- [Fixed a fatal error](https://github.com/Parsely/wp-parsely/issues/587) when requesting metadata for a post without categories and `categories as tags` enabled. [#588](https://github.com/Parsely/wp-parsely/pull/588)
+
```

Comment:
`develop` should have this content in it now they've been tagged, so this PR should be rebased onto `develop` so that it only includes the 3.1.0 changes.